Client/Frontend/Strings.swift

```diff
@@ -1471,5 +1471,5 @@ extension String {
 
     public static let TabTrayOtherTabsLabelTitle = MZLocalizedString("TabTray.OtherTabs.Title", value: "Other tabs", comment: "In the Tabs Tray, summoned from the homepage, the title for the section containing non-grouped tabs, which will appear below grouped tabs")
 
-    public static let FirefoxHomeJumpBackInSectionGroupSiteCount = MZLocalizedString("ActivityStream.JumpBackIn.TabGroup.SiteCount", value: "%d sites", comment: "On the Firefox homepage in the Jump Back In section, if a Tab group item - a collection of grouped tabs from a related search - exists, underneath the search term for the tab group, there will be a subtitle with how many tabs (or sites) are in that group. The placeholder is for a number. It will read '5 sites' or similar.")
+    public static let FirefoxHomeJumpBackInSectionGroupSiteCount = MZLocalizedString("ActivityStream.JumpBackIn.TabGroup.SiteCount", value: "Sites: %d", comment: "On the Firefox homepage in the Jump Back In section, if a Tab group item - a collection of grouped tabs from a related search - exists, underneath the search term for the tab group, there will be a subtitle with how many tabs (or sites) are in that group. The placeholder is for a number. It will read '5 sites' or similar.")
```

Comment:
```suggestion
    public static let FirefoxHomeJumpBackInSectionGroupSiteCount = MZLocalizedString("ActivityStream.JumpBackIn.TabGroup.SiteCount", value: "Tabs: %d", comment: "On the Firefox homepage in the Jump Back In section, if a Tab group item - a collection of grouped tabs from a related search - exists underneath the search term for the tab group, there will be a subtitle with a number for how many tabs are in that group. The placeholder is for a number. It will read 'Tabs: 5' or similar.")
    public static let FirefoxHomeJumpBackInSectionGroupTitle = MZLocalizedString("ActivityStream.JumpBackIn.TabGroup.Title", value: "Your search for \"%@\"", comment: "On the Firefox homepage in the Jump Back In section, if a Tab group item - a collection of grouped tabs from a related search - exists, the Tab Group item title will be 'Your search for \"video games\"'. The %@ sign is a placeholder for the actual search the user did.")
```